pyathena/converter.py

```diff
@@ -9,6 +9,7 @@
 from datetime import date, datetime, time
 from decimal import Decimal
 from typing import Any, Callable, Dict, Optional, Type
+from zoneinfo import ZoneInfo
```

Comment:
https://docs.python.org/3/library/zoneinfo.html
zoneinfo is not available in Python 3.8.